src/main/java/com/autotune/analyzer/services/UpdateResults.java

```diff
@@ -123,7 +122,8 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
                 LOGGER.error("updateResults API request payload for requestID {} failed", calCount);
                 sendErrorResponse(inputData, request, response, null, HttpServletResponse.SC_BAD_REQUEST, errorMessage);
             } else {
-                LOGGER.debug("updateResults API request payload for requestID {} success", calCount);
+                if (KruizeDeploymentInfo.log_http_req_resp)
```

Comment:
Hope partial failure like "out of 100 recods 20 saved " which is 400 response code

Reply:
Verified the response, no change in that.
Full API input is getting printed in logs in case of failure like this:

```
2024-05-1514:21:57.677 ERROR [qtp844112759-35][UpdateResults.java(122)]-updateResults API request payload for requestID 1 failed
2024-05-1514:21:57.682 INFO [qtp844112759-35][UpdateResults.java(167)]-UpdateRequestsAPI  input pay load [{"version":"v2.0","experiment_name":"quarkus-resteasy-kruize-min-http-response-time-db_0","interval_start_time":"2022-05-03T11:30:43.511Z","interval_end_time":"2022-05-03T11:45:43.602Z","kubernetes_objects":[{"type":"deployment","name":"tfb-qrh-deployment_0","namespace":"default_0","containers":[{"container_image_name":"kruize/tfb-qrh:1.13.2.F_et17","container_name":"tfb-server-0","metrics":[{"name":"cpuRequest","results":{"value":1.1,"format":"cores","aggregation_info":{"sum":4.4,"avg":1.1,"format":"cores"}}},{"name":"cpuLimit","results":{"value":0.5,"format":"cores","aggregation_info":{"sum":2.0,"avg":0.5,"format":"cores"}}},{"name":"cpuUsage","results":{"value":0.12,"format":"cores","aggregation_info":{"min":0.14,"max":0.84,"sum":0.84,"avg":0.12,"format":"cores"}}},{"name":"cpuThrottle","results":{"value":0.045,"format":"cores","aggregation_info":{"sum":0.19,"max":0.09,"avg":0.045,"format":"cores"}}},{"name":"memoryRequest","results":{"value":50.12,"format":"MiB","aggregation_info":{"sum":250.85,"avg":50.21,"format":"MiB"}}},{"name":"memoryLimit","results":{"value":100,"format":"MiB","aggregation_info":{"sum":500,"avg":100,"format":"MiB"}}},{"name":"memoryUsage","results":{"value":40.1,"format":"MiB","aggregation_info":{"min":50.6,"max":198.50,"sum":198.50,"avg":40.1,"format":"MiB"}}},{"name":"memoryRSS","results":{"aggregation_info":{"min":50.6,"max":123.6,"sum":123.6,"avg":31.91,"format":"MiB"}}}]},{"container_image_name":"kruize/tfb-qrh:1.13.2.F_et17","container_name":"tfb-server-1","metrics":[{"name":"cpuRequest","results":{"value":1.1,"format":"cores","aggregation_info":{"sum":4.4,"avg":1.1,"format":"cores"}}},{"name":"cpuLimit","results":{"value":0.5,"format":"cores","aggregation_info":{"sum":2.0,"avg":0.5,"format":"cores"}}},{"name":"cpuUsage","results":{"value":0.12,"format":"cores","aggregation_info":{"min":0.14,"max":0.84,"sum":0.84,"avg":0.12,"format":"cores"}}},{"name":"cpuThrottle","results":{"value":0.045,"format":"cores","aggregation_info":{"sum":0.19,"max":0.09,"avg":0.045,"format":"cores"}}},{"name":"memoryRequest","results":{"value":50.12,"format":"MiB","aggregation_info":{"sum":250.85,"avg":50.21,"format":"MiB"}}},{"name":"memoryLimit","results":{"value":100,"format":"MiB","aggregation_info":{"sum":500,"avg":100,"format":"MiB"}}},{"name":"memoryUsage","results":{"value":40.1,"format":"MiB","aggregation_info":{"min":50.6,"max":198.50,"sum":198.50,"avg":40.1,"format":"MiB"}}},{"name":"memoryRSS","results":{"aggregation_info":{"min":50.6,"max":123.6,"sum":123.6,"avg":31.91,"format":"MiB"}}}]}]}]},{"version":"v2.0","experiment_name":"quarkus-resteasy-kruize-min-http-response-time-db_0","interval_start_time":"2022-05-03T11:15:43.511Z","interval_end_time":"2022-05-03T11:30:43.602Z","kubernetes_objects":[{"type":"deployment","name":"tfb-qrh-deployment_0","namespace":"default_0","containers":[{"container_image_name":"kruize/tfb-qrh:1.13.2.F_et17","container_name":"tfb-server-0","metrics":[{"name":"cpuRequest","results":{"value":1.1,"format":"cores","aggregation_info":{"sum":4.4,"avg":1.1,"format":"cores"}}},{"name":"cpuLimit","results":{"value":0.5,"format":"cores","aggregation_info":{"sum":2.0,"avg":0.5,"format":"cores"}}},{"name":"cpuUsage","results":{"value":0.12,"format":"cores","aggregation_info":{"min":0.14,"max":0.84,"sum":0.84,"avg":0.12,"format":"cores"}}},{"name":"cpuThrottle","results":{"value":0.045,"format":"cores","aggregation_info":{"sum":0.19,"max":0.09,"avg":0.045,"format":"cores"}}},{"name":"memoryRequest","results":{"value":50.12,"format":"MiB","aggregation_info":{"sum":250.85,"avg":50.21,"format":"MiB"}}},{"name":"memoryLimit","results":{"value":100,"format":"MiB","aggregation_info":{"sum":500,"avg":100,"format":"MiB"}}},{"name":"memoryUsage","results":{"value":40.1,"format":"MiB","aggregation_info":{"min":50.6,"max":198.50,"sum":198.50,"avg":40.1,"format":"MiB"}}},{"name":"memoryRSS","results":{"aggregation_info":{"min":50.6,"max":123.6,"sum":123.6,"avg":31.91,"format":"MiB"}}}]},{"container_image_name":"kruize/tfb-qrh:1.13.2.F_et17","container_name":"tfb-server-1","metrics":[{"name":"cpuRequest","results":{"value":1.1,"format":"cores","aggregation_info":{"sum":4.4,"avg":1.1,"format":"cores"}}},{"name":"cpuLimit","results":{"value":0.5,"format":"cores","aggregation_info":{"sum":2.0,"avg":0.5,"format":"cores"}}},{"name":"cpuUsage","results":{"value":0.12,"format":"cores","aggregation_info":{"min":0.14,"max":0.84,"sum":0.84,"avg":0.12,"format":"cores"}}},{"name":"cpuThrottle","results":{"value":0.045,"format":"cores","aggregation_info":{"sum":0.19,"max":0.09,"avg":0.045,"format":"cores"}}},{"name":"memoryRequest","results":{"value":50.12,"format":"MiB","aggregation_info":{"sum":250.85,"avg":50.21,"format":"MiB"}}},{"name":"memoryLimit","results":{"value":100,"format":"MiB","aggregation_info":{"sum":500,"avg":100,"format":"MiB"}}},{"name":"memoryUsage","results":{"value":40.1,"format":"MiB","aggregation_info":{"min":50.6,"max":198.50,"sum":198.50,"avg":40.1,"format":"MiB"}}},{"name":"memoryRSS","results":{"aggregation_info":{"min":50.6,"max":123.6,"sum":123.6,"avg":31.91,"format":"MiB"}}}]}]}]},{"version":"v2.0","experiment_name":"quarkus-resteasy-kruize-min-http-response-time-db_0","interval_start_time":"2022-05-03T11:30:43.511Z","interval_end_time":"2022-05-03T11:45:43.602Z","kubernetes_objects":[{"type":"deployment","name":"tfb-qrh-deployment_0","namespace":"default_0","containers":[{"container_image_name":"kruize/tfb-qrh:1.13.2.F_et17","container_name":"tfb-server-0","metrics":[{"name":"cpuRequest","results":{"value":1.1,"format":"cores","aggregation_info":{"sum":4.4,"avg":1.1,"format":"cores"}}},{"name":"cpuLimit","results":{"value":0.5,"format":"cores","aggregation_info":{"sum":2.0,"avg":0.5,"format":"cores"}}},{"name":"cpuUsage","results":{"value":0.12,"format":"cores","aggregation_info":{"min":0.14,"max":0.84,"sum":0.84,"avg":0.12,"format":"cores"}}},{"name":"cpuThrottle","results":{"value":0.045,"format":"cores","aggregation_info":{"sum":0.19,"max":0.09,"avg":0.045,"format":"cores"}}},{"name":"memoryRequest","results":{"value":50.12,"format":"MiB","aggregation_info":{"sum":250.85,"avg":50.21,"format":"MiB"}}},{"name":"memoryLimit","results":{"value":100,"format":"MiB","aggregation_info":{"sum":500,"avg":100,"format":"MiB"}}},{"name":"memoryUsage","results":{"value":40.1,"format":"MiB","aggregation_info":{"min":50.6,"max":198.50,"sum":198.50,"avg":40.1,"format":"MiB"}}},{"name":"memoryRSS","results":{"aggregation_info":{"min":50.6,"max":123.6,"sum":123.6,"avg":31.91,"format":"MiB"}}}]},{"container_image_name":"kruize/tfb-qrh:1.13.2.F_et17","container_name":"tfb-server-1","metrics":[{"name":"cpuRequest","results":{"value":1.1,"format":"cores","aggregation_info":{"sum":4.4,"avg":1.1,"format":"cores"}}},{"name":"cpuLimit","results":{"value":0.5,"format":"cores","aggregation_info":{"sum":2.0,"avg":0.5,"format":"cores"}}},{"name":"cpuUsage","results":{"value":0.12,"format":"cores","aggregation_info":{"min":0.14,"max":0.84,"sum":0.84,"avg":0.12,"format":"cores"}}},{"name":"cpuThrottle","results":{"value":0.045,"format":"cores","aggregation_info":{"sum":0.19,"max":0.09,"avg":0.045,"format":"cores"}}},{"name":"memoryRequest","results":{"value":50.12,"format":"MiB","aggregation_info":{"sum":250.85,"avg":50.21,"format":"MiB"}}},{"name":"memoryLimit","results":{"value":100,"format":"MiB","aggregation_info":{"sum":500,"avg":100,"format":"MiB"}}},{"name":"memoryUsage","results":{"value":40.1,"format":"MiB","aggregation_info":{"min":50.6,"max":198.50,"sum":198.50,"avg":40.1,"format":"MiB"}}},{"name":"memoryRSS","results":{"aggregation_info":{"min":50.6,"max":123.6,"sum":123.6,"avg":31.91,"format":"MiB"}}}]}]}]}] 
```